chia/wallet/wallet_transaction_store.py

```diff
@@ -65,22 +65,19 @@ async def create(cls, db_wrapper: DBWrapper2):
             await conn.execute(
                 "CREATE INDEX IF NOT EXISTS tx_confirmed_index on transaction_record(confirmed_at_height)"
             )
-
             await conn.execute("CREATE INDEX IF NOT EXISTS tx_created_index on transaction_record(created_at_time)")
-
+            # Remove a redundant index on `created_at_time`
+            await conn.execute("DROP INDEX IF EXISTS tx_created_time")
```

Comment:
I would include the back-story of why we added this and why we're removing it. Perhaps even a link to the ticket https://github.com/Chia-Network/chia-blockchain/issues/10276